### ISSUE TITLE
why ip_adapter not work in controlnetimg2imgpipeline?

### ISSUE THREAD

**Issue by htwdls** [huggingface/diffusers#6157]
### Describe the bug

When using the controlnetimg2img pipe, id_adapter is not work.

### Reproduction

from diffusers import StableDiffusionControlNetImg2ImgPipeline, ControlNetModel
from diffusers import DDIMScheduler
import torch

controlnet = ControlNetModel.from_pretrained("./controlnet/", torch_dtype=torch.float16)
controlnet.load_state_dict(torch.load('./controlnet.ckpt'))

pipe = StableDiffusionControlNetImg2ImgPipeline.from_pretrained(
    "./stable-diffusion-v1-5/", controlnet=controlnet, torch_dtype=torch.float16, use_safetensors=False).to('cuda')

pipe.load_ip_adapter("./ip_adapter", subfolder="models", weight_name="ip-adapter_sd15.bin")

pipe.scheduler = DDIMScheduler.from_config(pipe.scheduler.config)
pipe.enable_xformers_memory_efficient_attention()
pipe.enable_model_cpu_offload()

# generate low res image
pose_image = Image.open('./img/pose.jpg')
prompt = "1girl,"
neg_prompt = "NSFW,watermarks,signature, text"
generator = torch.Generator('cuda').manual_seed(2001)
cloth_img = load_image("./img/1.png")
image = pipe(prompt=prompt, negative_prompt=neg_prompt,
             image=None, control_image=pose_image, strength=1.0, guidance_scale=7, generator=generator,
             num_inference_steps=25, height=512, width=768, control_guidance_start=0.0, control_guidance_end=1.0,
             ip_adapter_image=cloth_img).images[0]

### System Info

diffusers=0.24.0

**Comment by standardAI** [comment]
Hi @htwdls. `IP-Adapter` for `StableDiffusionControlNetImg2ImgPipeline` has just been added. You need to install `diffusers` from the source.
